package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "bootstrap": "^4.3.1",
```

Comment:
I found the bootstrap version we're using in the header of the `css/bootstrap.css`: `Bootstrap v4.0.0-alpha.6 (https://getbootstrap.com)` so we should be able to use that one https://www.npmjs.com/package/bootstrap/v/4.0.0-alpha.6

Reply:
I've revised this as specified. Cheers!!!